bciers/apps/reporting/tests/pages/operationsColumns.test.tsx

```diff
@@ -25,13 +25,66 @@ describe("operationColumns function", () => {
     );
     assert(columns[1].width === 560, "Column 2 width should be 560");
 
-    assert(columns[2].field === "more", 'Column 3 field should be "action"');
     assert(
-      columns[2].headerName === "More",
-      'Column 3 headerName should be "More"',
+      columns[2].field === "report_id",
+      'Column 3 field should be "report_id"',
+    );
+    assert(
+      columns[2].headerName === "Actions",
+      'Column 3 headerName should be "Actions"',
     );
-    assert(columns[2].sortable === false, "Column 3 sortable should be false");
     assert(columns[2].width === 120, "Column 3 width should be 120");
-    assert(columns[2].flex === 1, "Column 3 flex should be 1");
+
+    assert(columns[3].field === "more", 'Column 4 field should be "action"');
+    assert(
+      columns[3].headerName === "More",
+      'Column 4 headerName should be "More"',
+    );
+    assert(columns[3].sortable === false, "Column 4 sortable should be false");
+    assert(columns[3].width === 120, "Column 4 width should be 120");
+    assert(columns[3].flex === 1, "Column 4 flex should be 1");
+  });
+
+  it("has a 'start' button in the 'Actions' column when report_id is null", () => {
+    const columns: GridColDef[] = operationColumns();
+
+    const row = {
+      id: "1",
+      bcghg_id: "1",
+      name: "Operation without report",
+      report_id: null,
+      report_version_id: null,
+      report_status: null,
+    };
+
+    const params = {
+      row,
+      value: null,
+    };
+
+    const cell = columns[2].renderCell(params);
+    expect(cell.props.children).toBe("Start");
   });
+
+  it("has a 'continue' button in the 'Actions' column when report_id exists", () => {
+    const columns: GridColDef[] = operationColumns();
+
+    const row = {
+      id: "1",
+      bcghg_id: "1",
+      name: "Operation with report",
+      report_id: 1,
+      report_version_id: 1,
+      report_status: "draft",
+    };
+
+    const params = {
+      row,
+      value: 1,
+    };
+
+    const cell = columns[2].renderCell(params);
+    expect(cell.props.children).toBe("Continue");
```

Comment:
We could mock the router and assert that the right link has been clicked / URL called

https://github.com/bcgov/cas-registration/blob/develop/bciers/apps/administration/tests/components/contacts/Contacts.test.tsx#L6